doc/whats_new.rst

```diff
@@ -37,6 +37,8 @@ API
 
     - Now it is possible to plot only a subselection of channels in :func:`mne.viz.plot_raw` by using an array for order parameter by `Jaakko Leppakangas`_
 
+    - Added :class:`mne.decoding.gsoc.XdawnTransformer` transformer to work with two dimensional data array to comply with scikit-learn pipeline. Does not support ``overlap`` case by `Asish Panda`_
```

Comment:
@agramfort since there will be more than just one transformer, I think I can add all the new prototypes in `transformers.py` as they are relevant. What do you think?